packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -127,6 +127,7 @@ const processNodeUrl = (
   htmlPath: string,
   originalUrl?: string,
   server?: ViteDevServer,
+  isClassicScriptLink?: boolean,
```

Comment:
Maybe it could be `isModule` to avoid a few of the negations and have the same name we have in the other function?

Reply:
If we make this parameter `isModule`, the condition needs to be `isModule === false` because `undefined` means not a script here, and I felt that difficult to understand. How about `sciptType?: 'classic' | 'module' | null`?

Reply:
Then I think your original naming is good